docs/guide/release-notes/1.2.0.md

```diff
@@ -0,0 +1,75 @@
+---
+title: lit-html 1.2.0
+slug: lit-html-1-2-0
+tag: release-notes
+---
+
+See the [changelog](https://github.com/Polymer/lit-html/blob/master/CHANGELOG.md) for a complete list of changes.
+
+## `live` directive
+
+[Documentation](../template-reference#live)
+
+The `live` directive solves the problem of DOM state changing from underneath lit-html, for example with an `<input>` that a user can type into:
+
+```js
+html`<input .value=${live(x)}>`
+```
+
+In some cases you will want `x` to overwrite what the user has typed in. lit-html tracks the previous value of a binding and only updates the DOM is the value has changed. Without the live directive, lit-html might not think that the `value` property of the input has changed and won't update the DOM.
+
+The live directive ignores the previous value of the binding and always checks the current value in the DOM, causing the binding to update the input.
+
+We still reccomend that you handle user input so that the state and the DOM agree:
+
+```ts
```

Comment:
This example needs some explanation... I can't follow this at all. onInput returns a TemplateResult? how does that get rendered?

Reply:
Where does it re-render? 

I know this seems really simple to you, but I don't understand how this would fit into running code.  Doesn't it need a call to render somewhere?

I tried to put this into a jsbin and got nothing. https://jsbin.com/jesixadelo/edit?html,output

Reply:
Sorry, I forgot the render call inside go(): https://jsbin.com/qijedenulu/edit?html,output

Reply:
OK, I think there's a missing render around the template in `go()`?

Reply:
It all makes sense now 😂